Project.toml

```diff
@@ -13,6 +13,7 @@ Future = "9fa8497b-333b-5362-9e8d-4d0656e87820"
 IRTools = "7869d1d1-7146-5819-86e3-90919afe41df"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+LoopVectorization = "bdcacae8-1622-11e9-2a5c-532679323890"
```

Comment:
Why is this dependency added?

Reply:
vmap

Reply:
FWIW, NNlib and such have to use it for other reasons as well, so it's effectively becoming a common dependency for any Flux user already.

Reply:
It isn't referenced in the code anywhere?

Reply:
because I'm silly.

Reply:
It's used in NNlib, and the way we do things right now it to provide adjoints for whatever NNlib defines/needs here in Zygote.
Probably is better to go the other way around, remove NNlib's dependence from Zygote and have NNlib define its own adjoints, but this would be stuff for separate discussion and PR